tests/fertiscan/metadata/test_inspection.py

```diff
@@ -186,6 +182,44 @@ def test_missing_sub_label(self):
         self.assertListEqual(data["instructions"]["fr"], [])
         self.assertListEqual(data["instructions"]["en"], [])
 
+    def test_unequal_sub_label_lengths_and_order(self):
+        self.analyse["instructions_en"] = ["one"]
```

Comment:
you should instantiate `expected_instructions_en` and then 

```suggestion
        self.analyse["instructions_en"] = expected_instructions_en
```

This way we can 100% be certain of the order on the insert

Reply:
Sets allow handling such situations where the order might change: 
```python
        self.assertEqual(
            set(data["instructions"]["en"]),
            set(expected_instructions_en),
            "Instructions EN mismatch",
        )
```

I made the modification nonetheless.